cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,293 @@
+| content                 | Holds the source of truth to validate against the response of th get method or event.                     | true                                          | string, null, boolean or array |
+
+**Note:**
+
+- These are all the supported fields but depending on the test scenario and glue code used none of them are technically required.
+
+## Runtime Variables
+
+Runtime variables are used to store values from feature file and use them in dynamic firebolt object for making api calls, api validation, event validation etc.
+
+Below are the runtime variables that is used by fireboltCall object.
+
+- **attribute:** Attribute is a field name of the `runtime` environment variable that holds the value of the method name. For example, if the method name is `closedCaptions.setEnabled`, the attribute value will be `enabled`.
+- **value:** Value is a field name of the `runtime` environment variable that holds the value used for to set the value or for validation.
+
+**Note:** `resolveAtRuntime()` function is used by fireboltCall object in order to process runtime variables and other fields.
+
+## ResolveAtRuntime Function
+
+**ResolveAtRuntime** is a global function utilized by the JS files. This function is used to dynamically resolve values during test case execution. This function will replace variables in the fireboltCall object with the actual values from `runtime` variable.
+
+- **Input Types**: `resolveAtRuntime` accepts either a string or an array of strings as input.
+
+  - **String Input with Pattern**: When a string contains the '{{' pattern (e.g., `resolveAtRuntime("manage_closedcaptions.set{{attribute}}")`), the function replaces the pattern with the actual value from the `runtime` environment variable.
+  - **String Input without Pattern**: If the string does not contain the '{{' pattern (e.g., `resolveAtRuntime("value")`), the function directly returns the value from the `runtime` environment variable.
+  - **Array of Strings**: When an array of strings is provided, the function iterates over each string, replacing the patterns with the actual values from the `runtime` environment variable.
+
+- **Options:** The function has the support to accespts the optional parameter `uppercaseFirstChar` or `lowercaseFirstChar` to convert the first character of the resolved value to uppercase or lowercase. This parameter is separated by a dot from the attribute name. This parameter is used to convert the attribute value to the correct format for the method name. For example, `resolveAtRuntime('manage_closedcaptions.set{{attribute.uppercaseFirstChar}}')`.
+
+Usage of this function by dynamic firebolt object ensures that all necessary values are dynamically resolved during test case execution in the corresponding glue code.
+
+### Examples:
+
+Assuming runtime environment variable having below details
+
+```
+runtime: {
+ attribute: 'fontSize',
+ value: 1.5,
+}
+```
```

Comment:
Can you give a one liner to each Examples where resolveAtRuntime is called / 
Basically , How does resolveAtRuntime behave/return if we call this function by passing an array ?

Reply:
addressed the comments